### PR TITLE
Action issue fix

### PR DESCRIPTION
**Pull request by UttamSharma007**
Updated action and enqueue files function.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -33,19 +33,17 @@ function check_snippet_existence(){
 	
 	$type = get_post_meta($post->ID, '_bsf_post_type', true);
 	if($type){		
-		add_action( 'wp_head', 'frontend_style' );
 		add_action('wp_enqueue_scripts', 'enque');
 	}
 
 }
-function enque() {
+function enque( ) {
```
Comment (Nikschavan) [inline functions.php]: Add a prefix to function name to make this unique.

File: functions.php [hunk 2]
```diff
@@ -23,7 +23,7 @@
  * Initialize the metabox class.
  */
 /* FUNCTION to check for posts having snippets */
-add_action('wp','check_snippet_existence','',7);
+add_action( 'wp','check_snippet_existence' );
 function check_snippet_existence(){
```
Comment (Nikschavan) [inline functions.php]: Add a prefix to function name to make this unique

Comment (UttamSharma007) [reply functions.php]: Okay

File: functions.php (hunk 1 above)
Comment (UttamSharma007) [reply functions.php]: okay

**Pull request merged**